### PR TITLE
4.6.0

### PR DESCRIPTION
**Pull request by cdrini**
Review app: https://58-review-br-es6ify-b09jkq.archive.org/

After you push/merge, use this link to release: https://github.com/internetarchive/bookreader/releases/new?tag=v4.6.0&title=v4.6.0&body=Big%20change%3A%20all%20the%20plugins%20are%20now%20written%20in%20es6%2C%20and%20compiled%20to%20es5.%20There%20should%0Abe%20no%20observable%20changes%2C%20but%20note%20this%20was%20a%20big%20change.%0A%0A-%20Fix%3A%20UI%20on%20archive.org%20not%20consistent%20with%20demos%20due%20to%20CSS%20conflicts%3B%20%40rchrd2%0A-%20Convert%20all%20plugins%20to%20es6%20%2B%20webpack%3B%20%40cdrini%20%40iisa%20%40nsharma123%0A-%20Increase%20jest%20coverage%2039%25%20%E2%86%92%2047%25%3B%20%40cdrini%20%40iisa%20%40nsharma123%0A-%20Increase%20testcafe%20coverage%3B%20%40iisa%0A-%20Add%20testcafe%20to%20travis%3B%20%40cdrini%0A-%20Add%20more%2Fenforce%20more%20lint%20rules%3B%20%40cdrini%0A-%20Make%20demo%20pages%20more%20in%20sync%20with%20Internet%20Archive%20pages%3B%20%40mc2%0A-%20Add%20%60npm%20run%20serve-dev%60%20for%20auto-watching%20%2B%20auto-refreshing%20dev%20server%3B%20%40cdrini%0A

### CODE REVIEW

**Comment by cdrini** [comment]
Tested IE11:
- Freely available book: https://github.com/internetarchive/bookreader/blame/5c3e712203e5b6f91f665549e95c77952505fbad/BookReader/plugins/plugin.iframe.js#L65
    - ✅ 2up
        - ✅ click to flip
        - ✅ zoomed in drag to pan
        - ✅ zoomed in click to flip
    - ✅ 1up
        - ✅ drag to pan
    - ✅ thumb
        - ✅ drag to pan
    - ❌ search
        - ❌ 2up
            - ❌ searching when match on current page highlights said match
                - e.g. https://58-review-br-es6ify-b09jkq.archive.org/details/goodytwoshoes00newyiala/page/n5/mode/2up/search/margery
                - no errors in console
            - ✅ shows pins with tooltips containing keyword
            - ✅ Clicking pins goes to page and highlights keyword
            - ✅ When switching modes highlights still visible
            - ✅ searching for something else clears previous highlights/pins
        - ✅ 1up
            - ✅ searching when match on current page highlight said match
            - ✅ shows pins with tooltips containing keyword
            - ✅ Clicking pins goes to page and highlights keyword
            - ✅ When switching modes highlights still visible
            - ✅ searching for something else clears previous highlights/pins
        - ✅ thumb
            - ✅ searching when match on current page highlight said match
            - ✅ shows pins with tooltips containing keyword
            - ✅* Clicking pins goes to page and highlights keyword
                - Nope; goes to page but does not highlight. pre-existing
            - ✅* When switching modes highlights still visible
                - Not when switching to thumb mode; pre-existing bug though
            - ✅ searching for something else clears previous highlights/pins

**Comment by cdrini** [comment]
Failing test fixed

File: BookReader/BookReader.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ function BookReader(options) {
     this.setup(options);
 }
 
-BookReader.version = '4.5.1';
+BookReader.version = '4.6.0';
```
Comment (iisa) [inline BookReader/BookReader.js]: 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
+# 4.6.0
+Big change: all the plugins are now written in es6, and compiled to es5. There should
```
Comment (iisa) [inline CHANGELOG.md]: 👏

**Pull request merged**